plugins/vendor/zoom/ZoomDropFolderPlugin/batch/processors/zoomRecordingProcessor.php

```diff
@@ -114,10 +114,11 @@ protected function handleVideoRecord($recording, $ownerId, $validatedUsers, $val
 		$flavorAsset = KBatchBase::$kClient->flavorAsset->add($entry->id, $kFlavorAsset);
 		
 		$resource = new KalturaUrlResource();
-		list($redirectUrl, $urlHeaders) = $this->getZoomRedirectUrlFromFile($recording);
-		$resource->url = $redirectUrl;
+        $urlHeaders = $this->getZoomAuthorizationHeaderFromFile($recording);

```

Comment:
getZoomAuthorizationHeaderFromFile returns array, why are you assigning to a variable?